andela/AndelaEatsUI
# #160392895 View orders on admin dashboard

#### What does this PR do?
- Adds static UI template for viewing orders on the admin dashboard 
#### Description of Task to be completed?
- Create orders component for the admin dashboard
- Configure component navigation
#### How should this be manually tested?
- Log in to the app
- Switch to the admin dashboard from the user profile drop-down menu or [here](http://http://andelaeats-dev.andela.com:3000/admin/orders)
- You should see a static page displaying orders
#### What are the relevant pivotal tracker stories?
- [#160392895](https://www.pivotaltracker.com/story/show/160392895)
#### Background Context
* This is a UI only task, the functionality will be added in a separate story
#### Screenshots
![image](https://user-images.githubusercontent.com/26303683/45372291-84346f80-b5e4-11e8-8db6-9b9ddaabe54b.png)
